runtime-testsuite/src/org/antlr/v4/testgen/TargetConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package org.antlr.v4.testgen;
```

Comment:
The problem with the approach isn't that it won't work. My specific concerns are the following:

1. The runtime test suite is currently *fully* target-independent. Keeping it that way increases its maintainability and generality.
2. The functionality added by this change already exists. If it's not working as expected (or as needed), we should address the reasons why it's not working in practice rather than add a second piece of code to do the same thing.